src/System.Text.Primitives/System/Text/Formatters/Utf16.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Text.Formatters
+{
+    public static partial class Utf16
+    {
+        #region Common constants
+
+        private const char Colon = ':';
```

Comment:
That's not to say that it was the "right" way to break it up. They are all private anyways, so we can move them however we want.